launcherctl/_launcher.py

```
@@ -50,23 +48,23 @@ def is_active(self) -> bool:
 
 class API:
     def keys(self) -> list[str]:
-        return [x.decode("utf-8") for x in launcherctl("list-launchers").splitlines()]
+        return [x for x in launcherctl("list-launchers").splitlines()]
```

Ruff (0.15.11)

[warning] Unnecessary list comprehension (rewrite using `list()`)

Rewrite using `list()`

(C416)